FreeTubeApp/FreeTube
# Update app menu to add items for going back/forward in router history

# Update app menu to add items for going back/forward in router history




## Pull Request Type


## Related issue
For #1869

## Description
Update app menu to add items for going back/forward in router history

## Screenshots 
![image](https://user-images.githubusercontent.com/1018543/206338962-16effb39-80d7-4a83-aeff-26be9822ce25.png)

## Testing 
- Open app
- Navigate to another view
- Use menu item
- [Existing keyboard shortcuts](https://docs.freetubeapp.io/usage/keyboard-shortcuts/) should still work (Alt + Left/Right Arrow)

## Desktop

- **OS:** macOS
- **OS Version:** 12.6.1
- **FreeTube version:** 7abe97d700695525b2c5d4af95a561dec4ca640f

## Additional context
Does that mean we can move the implementation of keyboard shortcut to the new menu item?
